src/Components/Server/test/Circuits/CircuitRegistryTest.cs

```diff
@@ -316,39 +316,6 @@ public async Task DisconnectWhenAConnectIsInProgress()
             Assert.False(registry.DisconnectedCircuits.TryGetValue(circuitHost.CircuitId, out _));
         }
 
-        [Fact]
-        [Flaky("https://github.com/aspnet/AspNetCore-Internal/issues/2689", FlakyOn.All)]
-        public void CircuitRegistryUsesConfiguredMaxRetainedDisconnectedCircuitsValue()
```

Comment:
Do we have this functionality covered in some other test?

Reply:
We're effectively testing `MemoryCache`'s implementation here, it's not really useful.